src/unlimited.cpp

```diff
@@ -1153,7 +1153,16 @@ bool IsChainNearlySyncd()
     LOCK(cs_ischainnearlysyncd);
     return fIsChainNearlySyncd;
 }
-
+uint64_t LargestBlockSeen(uint64_t nBlockSize)
+{
+     // return the largest block size that we have seen since startup
+     static uint64_t nLargestBlockSeen = BLOCKSTREAM_CORE_MAX_BLOCK_SIZE;
```

Comment:
Blockstream has no "preferred blocksize" and "blockstream core" isn't something that exists, except as a really nasty insult by a collection of dishonest people who are incorrectly claiming that the work of the Bitcoin project's is somehow owned by blockstream because a couple blockstream employees contributed too.